argus/client/tunnel_ssh.py

```
@@ -0,0 +1,367 @@
+        process.terminate()
+        try:
+            process.wait(timeout=5)
+        except subprocess.TimeoutExpired:
+            process.kill()
+            process.wait(timeout=5)
+
+
+def scan_host_keys(host: str, port: int) -> list[str]:
+    try:
+        result = subprocess.run(  # noqa: S603
+            ["ssh-keyscan", "-T", "5", "-p", str(port), "-t", "ed25519,ecdsa", host],
```

Ruff (0.15.14)

[error] Starting a process with a partial executable path

(S607)